src/utils/EventBus.ts

```diff
@@ -0,0 +1,31 @@
+export class EventBus {
+  private readonly listeners: Record<string, Array<() => void>> = {};
```

Comment:
Обработчик события может принимать в себя аргументы
```suggestion
  private readonly listeners: Record<string, Array<(...args: unknown[]) => void>> = {};
```